Have testing overhead dashboard only use successful workflows

PaliC opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #100580

Once this pr is approved I'll update the data to do this.

Relevant queryhttps://console.rockset.com/query?query_text_id=8b05b37e-747b-421c-9347-151586b0ac80

PaliC committed:
Have testing overhead dashboard only use successful workflows

[ghstack-poisoned]

PaliC committed:
Update on "Have testing overhead dashboard only use successful workflows"


Once this pr is approved I'll update the data to do this.

Relevant query https://console.rockset.com/query?query_text_id=1d9e0d35-9ae3-434f-bf18-649f6bc62a1c

[ghstack-poisoned]

PaliC commented:
@pytorchbot merge

ZainRizvi reviewed (changes requested):
Loooks good for the most part, but the workflow linked doesn't seem to specify a specific repo. The rockset query contains data from many repos beyond just pytorch/pytorch.

You probably want to add the repo owner & name as an additional filter here

PaliC committed:
Update on "Have testing overhead dashboard only use successful workflows"


Once this pr is approved I'll update the data to do this.

Relevant query https://console.rockset.com/query?query_text_id=d67ad12b-6714-4181-95ff-50670e2f645e

[ghstack-poisoned]

PaliC commented:
@pytorchbot merge -r

PaliC committed:
Update on "Have testing overhead dashboard only use successful workflows"


Once this pr is approved I'll update the data to do this.

Relevant queryhttps://console.rockset.com/query?query_text_id=8b05b37e-747b-421c-9347-151586b0ac80

[ghstack-poisoned]
